### DISCUSSION TITLE
Weely Hours List Not Combining Data

### DISCUSSION THREAD

**Discussion by jrpetersjr** [kimai/kimai#5127] [General]
When looking at the weekly hours list I see that all of my data is split out into different items even though they are the same project and the same activity. You can see it for the first 3. The Activities are global activities because I want to use them for all of the projects currently, but I'm not seein it. I should be able to see only 2 items in the list. 1 for the first 3 totally 6:50 and 1 for the last totally 1 hour. Am I missing setup for this?

![image](https://github.com/user-attachments/assets/bc2b6cdc-a90f-4469-87be-6f57947d04d4)

**Comment by kevinpapst** [comment]
merging is not always possible. share a screenshot of my times.

**Reply by jrpetersjr** [reply]
Why would these not be able to be combined? It's the same project and the same task. Maybe I'm looking in the wrong spot for what I'm looking for.
